ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/InMemory.hs

```diff
@@ -262,19 +282,39 @@ class Monad m => ResolvesBlocks m blk | m -> blk where
 instance Monad m => ResolvesBlocks (ReaderT (ResolveBlock m blk) m) blk where
   resolveBlock r = ReaderT $ \f -> f r
 
+-- Quite a specific instance so we can satisfy the fundep
```

Comment:
Should we clarify which fundep we're referring to?

Reply:
:man_shrugging: I just moved this instance from some lines below.